Blish HUD/_Extensions/ColorExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+
+namespace Blish_HUD._Extensions
+{
+    public static class ColorExtensions
+    {
+        public static Microsoft.Xna.Framework.Color ToXnaColor(this Gw2Sharp.WebApi.V2.Models.ColorMaterial color)
+        {
+            return  new Color(color.Rgb[0], color.Rgb[1], color.Rgb[2]);
+        }
+    }
+}
```

Comment:
Would prefer K&R style for consistency.  Sorry for the editorconfig not being the right one!